src/app/find/mapPage/MapPage.jsx

```diff
@@ -162,6 +162,7 @@ export default class MapPage extends Component {
       radius: Math.floor(radius),
       minResults,
       searchString,
+      locationFieldsOnly: true,
```

Comment:
Wasn't this change already made in the last PR?

Reply:
I think I forgot to make that PR for the streetlives-web repo :x so here it is